resource/reactivedrop_german.txt

```diff
@@ -7,11 +7,11 @@
 "[english]asw_SynTekMegacorp"		"SynTek Megacorporation, Inc."
 "asw_SynTekPDA"		"SynTek Persönlicher Digitaler Assistant"
 "[english]asw_SynTekPDA"		"SynTek Personal Digital Assistant"
-"asw_SynTekAccount"		"Besitzer: %s1"
+"asw_SynTekAccount"		"Benutzer: %s1"
```

Comment:
Ich habe das absichtlich in "Besitzer" geändert weil der Benutzer ja die Person sein könnte die den PDA gerade aufgehoben hat. Dementsprechend würde ich das gerne nicht auf "Benutzer" ändern.